django/core/serializers/pyyaml.py

```diff
@@ -44,17 +44,17 @@ class Serializer(PythonSerializer):
 
     internal_use_only = False
 
-    def handle_field(self, obj, field):
+    def _value_from_field(self, obj, field):
```

Comment:
Here's the superclass's implementation of `_value_from_field`: https://github.com/django/django/blob/40bfd7b09aa0907b143e96f0b055538f476e544e/django/core/serializers/python.py#L41-46